app/components/subheader/schema.yml

```diff
@@ -1,10 +1,33 @@
 _description: |
   A simple subheader component.
 
+type:
+  _label: Type
+  _has:
+    input: radio
+    options:
+      - name: "small (h4)"
+        value: "h4"
+      - name: "medium (h3)"
+        value: "h3"
+      - name: "large (h2)"
+        value: "h2"
```

Comment:
Remember to change double quotes for single quotes. Another alternative would be to remove the quotes and leave it as is. That's also valid in `.yml` files